src/Presentation/HtmlMappingsPresenter.php

```diff
@@ -38,8 +39,8 @@ public function showMappings( MappingList $mappingList, bool $canEdit ): void {
 
 	private function createHeader(): string {
 		return '<thead id="wikibase-rdf-header"><tr>'
-			. '<th class="wikibase-rdf-mappings-predicate-heading">' . wfMessage( 'wikibase-rdf-mappings-predicate-heading' ) . '</th>'
-			. '<th class="wikibase-rdf-mappings-object-heading">' . wfMessage( 'wikibase-rdf-mappings-object-heading' ) . '</th>'
+			. '<th class="wikibase-rdf-mappings-predicate-heading">' . $this->msg( 'wikibase-rdf-mappings-predicate-heading' ) . '</th>'
+			. '<th class="wikibase-rdf-mappings-object-heading">' . $this->msg( 'wikibase-rdf-mappings-object-heading' ) . '</th>'
```

Comment:
You are not calling any function on the object. Not sure what __toString does. Since the values are put into HTML without escaping, you probably should be calling ->escaped(). I doubt __toString is the same, and being explicit about escaping is nice.